src/app/grants/mainGrant.tsx

```diff
@@ -15,11 +14,41 @@ export default function MainGrant({
   return (
     <div className="flex flex-col">
       <GrantsHero />
-      <Eligibility />
-      <StartGrant setPage={setPage} />
-      <Perks />
-      <Cohort setPage={setPage} />
-      <CohortStructure />
+      <BtrustGrant />
+      {/* <Eligibility /> */}
+      <AllGrants setPage={setPage} />
+      <FaqContainer
+        faqs={[
+          {
+            answer:
+              "Btrust Grants provide financial support to developers, educators, and event organizers working to strengthen the Bitcoin open-source ecosystem. We offer grants for developer contributions, education programs, research, and Bitcoin-only events to remove financial barriers and enable impactful contributions.",
+            question: "What are Btrust Grants?",
+          },
+          {
+            answer:
+              "Our grants are open to developers, educators, and organizers from Africa, India, Latin America, MENA, and Southeast Asia. We also consider applicants from other Global South regions on a case-by-case basis, contingent on their Proof of Work.",
+            question: "Who can apply for a Btrust Grant?",
+          },
+          {
+            answer: "We fund:",
+            question: "What kind of projects do you fund?",
+            lists: [
+              "Open-source Bitcoin development and research.",
```

Comment:
Take out "research" for now

```suggestion
              "Open-source Bitcoin development.",
```